### PR TITLE
Rename packages organization from `com.velocidi` to `com.kevel`

### PR DESCRIPTION
**Pull request by beatriz**
This changes the packages from the `com.velocidi` to `com.kevel` organization to reflect the organizational change. For now, it doesn't touch the publishing organization as I still need to figure out some things for that.

After making the change I wondered if we'd want to use `adzerk` instead to be consistent with the GitHub organization name, what do you think?

I am leaving this with the `do not merge label` for now, as I know we'd like to get a Pekko update without this impacting its release. I will also try to figure out the publish organization to try to get it all at once.

### Does this change relate to existing issues or pull requests?

No.

### Does this change require an update to the documentation?

Updated the `README` docs as well.

### How has this been tested?

Relying on CI to validate everything keeps working.

### CODE REVIEW

**Comment by jcazevedo** [comment]
> After making the change I wondered if we'd want to use `adzerk` instead to be consistent with the GitHub organization name, what do you think?

I'd rather not move from one old name (`com.velocidi`) to another old one (`com.adzerk`).

File: hashing/src/main/java/com/velocidi/apso/hashing/MurmurHash3.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-package com.velocidi.apso.hashing;
+package com.kevel.apso.hashing;
```
Comment (jcazevedo) [inline hashing/src/main/java/com/velocidi/apso/hashing/MurmurHash3.java:24]: Can we rollback the other changes in this file? I suspect some unwanted IDE shenanigans went unnoticed here.

**Review by jcazevedo — CHANGES_REQUESTED** [review]
Thanks for getting the ball rolling on this!

Can we also rename the `**/com/velocidi/*` folders to `**/com/kevel/*`?

**Comment by beatriz** [comment]
> Can we also rename the `**/com/velocidi/*` folders to `**/com/kevel/*`?

Yeah, totally missed that 😅 Done!

**Review by jcazevedo — APPROVED** [review]
Thank you!

**Pull request merged**